### PR TITLE
PR: Cancel AFK on Chat

### LINKED ISSUE #3353: AFK not cancelled on chat event when "cancel-afk-on-interact" is false
### Information

**Full output of `/ess version`**:
```
[21:15:09 INFO]: Server version: 1.15.2-R0.1-SNAPSHOT git-Paper-333 (MC: 1.15.2)
[21:15:09 INFO]: EssentialsX version: 2.17.2.45
[21:15:09 INFO]: Vault is not installed. Chat and permissions may not work.
```

**Server log**: not needed
**EssentialsX config**: not needed

### Details

**Description**  
I think this behavior should be changed.
Currently, if `cancel-afk-on-move: true` and `cancel-afk-on-interact: false` in the config, the player will be leave AFK state on move, but not for chat, despite the comments in the Ess config implying otherwise

**Steps to reproduce**  
Set two config values as specified above

**Expected behavior**  
Interact event cancel includes clicking blocks etc, but not chat

Possibly could also rephrase this GitHub Issue as a feature request instead, to separate out a config setting for `cancel-afk-on-chat-or-command` to distinguish from `cancel-afk-on-interact`... thoughts?

### PR DESCRIPTION
**Pull request by AstralisVox**
I have made it so there is now an option to cancel a players afk status when they send a normal chat message. I have added an option to the config file for this as well. As I felt, it would be better a separate option, instead of using one of the current options.

REF: #3353

### CODE REVIEW

File: Essentials/src/com/earth2me/essentials/Settings.java [hunk 1]
```diff
@@ -24,17 +24,8 @@
...
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
+import java.util.*;
```
Comment (Laarryy) [inline Essentials/src/com/earth2me/essentials/Settings.java]: Don't use `*` imports - I know IntelliJ does it automatically but you should turn it off in this case

File: Essentials/src/com/earth2me/essentials/User.java [hunk 2]
```diff
@@ -25,13 +25,7 @@
...
 
 import java.math.BigDecimal;
-import java.util.Calendar;
-import java.util.GregorianCalendar;
-import java.util.List;
-import java.util.Locale;
-import java.util.Map;
-import java.util.UUID;
-import java.util.WeakHashMap;
+import java.util.*;
```
Comment (Laarryy) [inline Essentials/src/com/earth2me/essentials/User.java]: Don't use `*` imports - I know IntelliJ does it automatically but you should turn it off in this case

File: Essentials/src/com/earth2me/essentials/User.java [hunk 3]
```diff
@@ -31,7 +31,7 @@
 import java.util.Locale;
 import java.util.Map;
 import java.util.UUID;
-import java.util.WeakHashMap;
+import java.util.WeakHashMap;;
```
Comment (Chew) [inline Essentials/src/com/earth2me/essentials/User.java]: What happened here

Comment (pop4959) [reply Essentials/src/com/earth2me/essentials/User.java]: Really minor but I fixed it for him

Comment (Chew) [reply Essentials/src/com/earth2me/essentials/User.java]: Epic

**Review by Octol1ttle — DISMISSED** [review]

**Comment by JRoy** [comment]
Hello, can you please rebase this pull request so we can merge it! Thanks!

**Comment by JRoy** [comment]
hey @OmegaWeaponDev I saw your recent commit but wanted to let you know you still didn't rebase (or merge) your pull request on top of `2.x`

**Comment by AstralisVox** [comment]
> hey @OmegaWeaponDev I saw your recent commit but wanted to let you know you still didn't rebase (or merge) your pull request on top of `2.x`

Hmm okay I thought I did as I did press rebase. I will try a merge soon and see how we go.

**Pull request closed without merging**